torchgeo/datamodules/pastis.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""PASTIS datamodule."""
+
+from typing import Any
+
+import kornia.augmentation as K
+import torch
+from torch import Tensor
+from torch.utils.data import random_split
+
+from ..datasets import PASTIS
+from .geo import NonGeoDataModule
+
+
+def collate_fn(
```

Comment:
This should be in `torchgeo/datasets/utils.py` with the other collation functions so we can easily reuse it across multiple datamodules.